gcp/gke/README.md

```diff
@@ -0,0 +1,44 @@
+# GKE
+Mozilla IT Service Engineering team module for creating a GKE cluster
+
+This module wraps the upstream terraform registry [GKE module](https://registry.terraform.io/modules/terraform-google-modules/kubernetes-engine/google/) with some additional niceties
+
+Its worth noting that we are using the beta version of the module to take advantage of the workload identity feature
+
+## Usage:
+Sample usage
+
+```hcl
+locals {
+  project_id   = "mozilla-it-service-engineering"
+  cluster_name = "test-cluster"
+
+  node_pools = [
+    {
+      name               = "default-np-1"
+      machine_type       = "e2-small"
+      min_count          = "3"
+      max_count          = "10"
+      max_surge          = "3"
+      auto_repair        = true
+      auto_upgrade       = true
+      initial_node_count = 2
+
+    }
+  ]
+}
+
+module "gke" {
+  source       = "github.com/mozilla-it/terraform-modules//gcp/gke?ref=master"
+  costcenter   = "1410"
+  environment  = "dev"
+  project_id   = local.project_id
+  cluster_name = local.cluster_name
```

Comment:
"name" would be more standards friendly than "cluster_name".

Reply:
Fixed in 7b1b3f5c949de5f14cb306d2e68b26350b63ff8e